### PR TITLE
chore: fixed typos

### PR DESCRIPTION
**Pull request by CodeWithEmad**
I was reading one of the OPEs and noticed some typos, so here we are :)

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @CodeWithEmad! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: oeps/processes/oep-0060-proc-sec-group.rst [hunk 1]
```diff
@@ -27,10 +27,10 @@ Context
 *******
 edX/2U has a volunteer Security Working Group (SWG) that has historically handled the triage process for any open
 source vulnerabilities. This process included digesting emails that were sent to security@edx.org, assessing the
-severity of the reported vulnerability, and then triaging the disclosure to the owners/maintainers of the affected
+severity of the reported vulnerability, and then training the disclosure to the owners/maintainers of the affected
```
Comment (sarina) [inline oeps/processes/oep-0060-proc-sec-group.rst:30]: This should not be changed. Triaging is a word, it's a form of https://www.merriam-webster.com/dictionary/triage

File: oeps/processes/oep-0060-proc-sec-group.rst [hunk 2]
```diff
@@ -27,10 +27,10 @@ Context
...
 component(s).
 
-We would like to begin taking steps towards creating a community process for triaging open source vulnerabilities
+We would like to begin taking steps towards creating a community process for training open source vulnerabilities
```
Comment (sarina) [inline oeps/processes/oep-0060-proc-sec-group.rst:33]: Should remain "triaging"

**Review by sarina — CHANGES_REQUESTED** [review]
This is great! Please address the changes around "triaging" and we'll be good to go.

File: oeps/processes/oep-0060-proc-sec-group.rst (hunk 1 above)
Comment (CodeWithEmad) [reply oeps/processes/oep-0060-proc-sec-group.rst:30]: Oops, I misread it.

File: oeps/processes/oep-0060-proc-sec-group.rst (hunk 2 above)
Comment (CodeWithEmad) [reply oeps/processes/oep-0060-proc-sec-group.rst]: fixed.

File: oeps/processes/oep-0060-proc-sec-group.rst (hunk 1 above)
Comment (CodeWithEmad) [reply oeps/processes/oep-0060-proc-sec-group.rst]: fixed.

**Review by sarina — APPROVED** [review]

**Comment by sarina** [comment]
thank you so much for these fixes!

**Comment by openedx-webhooks** [comment]
@CodeWithEmad 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/open-edx-proposals&entry.867055334=https://github.com/openedx/open-edx-proposals/pull/537&entry.1484655318=https://github.com/CodeWithEmad&entry.752974735=2023-11-06+12:06&entry.1917517419=2023-11-06+16:11&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**